index.d.ts

```diff
@@ -217,6 +217,28 @@ declare module '@primer/components' {
 
   export const Link: React.FunctionComponent<LinkProps>
 
+  export type PaginationHrefBuilder = (page: number) => string
+
+  export type PaginationPageChangeCallback = (e: React.MouseEvent, page: number) => void
+
+  export interface PaginationProps extends CommonProps {
+    currentPage: number
+    hrefBuilder?: PaginationHrefBuilder
+    /**
+     * How many pages to show on the left and right of the component
```

Comment:
Do these comments show up in VSCode when someone is writing TS? That's pretty sweet

Reply:
Yeah they power TS intellisense IIRC